Server/Server.Core/Server.Core.Static/Server.cpp

```diff
@@ -1,13 +1,78 @@
 #include <future>
 
 #include "Server.hpp"
-#include "DataAccess.Postgre/PostgreRepositoryManager.hpp"
+
 
 namespace Server
 {
 using namespace DataAccess;
 using UtilityTime::safe_localtime;
 
+
+
+Server::Server() { /* do nothing*/ }
+
+
+void Server::initRepository(std::unique_ptr<PostgreRepositoryManager> postgreManager)
+{
+    _postgreManager.swap(postgreManager);
+}
+
```

Comment:
We can't just initialize these values without a constructor. These class members must be initialized first, so that we can change them later if necessary

Reply:
The server could be constructed using an already existing object. That's the idea of builder pattern. Other question why am I used here qualified type instead of interface type. That is wrong...